addons/zeus/functions/fnc_playSoundZeus.sqf

```diff
@@ -24,20 +24,53 @@ private _onConfirm =
 	];
 	//Get in params again
 	_in params [["_pos",[0,0,0],[[]],3], ["_unit",objNull,[objNull]]];
+
+	// Clear sound preview
+	stopSound GVAR(soundPreview);
+	GVAR(soundPreviewSelected) = nil;
+	GVAR(soundPreview) = nil;
 	
 	// manage input on _tarets to get one list
 	_targets = [_targets] call FUNC(getListZenOwnersSelection);
 
+	_soundIDList = [];
 	if (count _targets == 0) then {
 		// play sound
-		[_pos, _range, _sound, _volume, false] call EFUNC(sounds,playSoundPos);
+		private _soundID = ([_pos, _range, _sound, _volume, false] call EFUNC(sounds,playSoundPos));
+		_soundIDList pushBack _soundID;
 	} else {
 		// send event
 		[QEGVAR(sounds,playSoundLocal), [_pos, _range, _sound, _volume, true], _targets] call CBA_fnc_targetEvent;
+		// TODO: how to get soundID from this?
+	};
+
+	// Attach the sound ID list to a logic object, so it can be deleted mid-play
+	_logicCenter = createCenter sideLogic;
+	_logicGroup = createGroup _logicCenter;
+	_logic = _logicGroup createUnit ["Logic", _pos, [], 0, "NONE"];
```

Comment:
just double checking, is this visible to all other zeus'? I believe ``createVehicle`` should be, but can't remember if ``logic`` have specific behaviour compared to normal units?

Reply:
Think I borrowed the code from the fireSupport module, so....should be visible to all zeuses